omod/src/main/webapp/resources/htmlforms/Covid19-PostMortem.xml

```diff
@@ -0,0 +1,248 @@
+                                        <label>Evidence of Medical Intervention</label>
+                                        <obs id="1542" conceptId="1542"
+                                             answerConceptIds="166272,159466,166273,5622"
+                                             answerLabels="cannula, I.V line, NGT, catheters, Other Specify"/>
+                                    </div>
+                                    <div class="col-md-4">
+                                        <label>Other Medical Intervention</label>
+                                        <obs id="165136" conceptId="165136"/>
+                                    </div>
+                                    </div>
+                                    <div class="row">
+                                        <obs conceptId="90251" answerConceptId="871" answerLabel="Pallor" style="checkbox" class="col-md-2"/>
+                                        <obs conceptId="90251" answerConceptId="90132" answerLabel="Jaundice" style="checkbox" class="col-md-2" />
+                                        <obs conceptId="90251" answerConceptId="158843" answerLabel="Adenopathy" style="checkbox" class="col-md-2" />
+                                        <obs conceptId="90251" answerConceptId="165501" answerLabel="Skin lesions (rash, bleeding etc.)" style="checkbox" class="col-md-2"/>
+                                        <obs conceptId="90251" answerConceptId="90094" answerLabel="Bleeding through orifices" style="checkbox" class="col-md-3" />
+                                        <obs conceptId="90251" answerConceptId="5980" answerLabel="Oedema" style="checkbox" class="col-md-3"/>
+                                        <obs id="90002" conceptId="90251" answerConceptId="90002" answerLabel="Other" style="checkbox" class="col-md-2" />
+                                        <div class="col-md-4">
+                                            <label>Other Significant Finding</label>
+                                            <obs id="166296" conceptId="166296" />
+                                        </div>
+                                    </div>
+                                    <div class="row">
+                                    <div class="col-md-4">
+                                        <label>Cause of Death</label>
+                                        <obs id="5002" conceptId="5002"
+                                             answerConceptIds="90001,99288,99287,58,165374,165375,165376,165377,116030"
+                                             answerLabels="UNKNOWN, DISEASE, ACCIDENT, TUBERCULOSIS, Infectious Disease other than TB, HIV Non-infectious Disease, Natural Causes, Non natural causes, Cancer"/>
+                                    </div>
+                                    <div class="col-md-4">
+                                        <label>Medical Officer</label>
```

Comment:
Use provider tag to provide a drop of all provider

Reply:
@semujju Done